src/System.Private.CoreLib/shared/System/Numerics/Vector.cs

```diff
@@ -809,6 +867,52 @@ public Vector(Span<T> values)
         #endregion Constructors
 
         #region Public Instance Methods
+        /// <summary>
+        /// Copies the vector to the given <see cref="Span{Byte}"/>. The destination span must be at least size <see cref="Vector{Byte}.Count"/>.
+        /// </summary>
+        /// <param name="destination">The destination span which the values are copied into</param>
+        /// <exception cref="ArgumentException">If number of elements in source vector is greater than those available in destination span</exception>
+        public void CopyTo(Span<byte> destination)
```

Comment:
What does the codegen look like for these methods currently?

Reply:
I'm working on getting the disasm and will post it in the main comments.